test/ctest_local_replication

```diff
@@ -0,0 +1,67 @@
+#
+# Local replication test for MongoDB image.
+#
+# Always use sourced from a specific container testfile
+
+function ctest_local_replication() {
```

Comment:
`cleanup` function also contained 

```bash
local network_name="mongodb-replset-$$"
! docker network ls | grep -q ${network_name} || docker network rm ${network_name}
```

to remove network created for this test.

So I suggest to add something like this into this function
```bash
function cleanup() {
  ct_cleanup()
  local network_name="mongodb-replset-$$"
  ! docker network ls | grep -q ${network_name} || docker network rm ${network_name}
}
trap cleanup EXIT SIGINT
```

and to the end of this function add `ct_enable_cleanup`